README.md

```diff
@@ -229,6 +229,29 @@ Number of people named "Alice": 1
 Alice
 ```
 
+### Running an Upsert
+
+Txn#mutate(Mutation) runs mutations and upsert transactions. It takes in a Mutation object, which provides two data formats: JSON and RDF N-Quad. You can choose whichever way is convenient.
```

Comment:
This doesn't seem right. @prashant-shahi please have a discussion about this with me. Something seems off.

Reply:
Please DO NOT MERGE.